Base/QTCore/qSlicerExtensionsManagerModel.cxx

```diff
@@ -1714,13 +1714,14 @@ bool qSlicerExtensionsManagerModel::downloadAndInstallExtensionByName(const QStr
 }
 
 //-----------------------------------------------------------------------------
-void qSlicerExtensionsManagerModel::installExtensionFromServer(const QString& extensionName, bool restart)
+bool qSlicerExtensionsManagerModel::installExtensionFromServer(const QString& extensionName, bool restart)
 {
   Q_D(qSlicerExtensionsManagerModel);
 
   if (this->isExtensionInstalled(extensionName))
     {
-    return;
+    // Already installed; nothing to do
```

Comment:
Maybe we check if an update is available for an extension and if a flag is specified that the latest version of the extension should be installed then we should install that update.

Reply:
Now tracker in https://github.com/Slicer/Slicer/issues/7149